notebooks/04.02-Simple-Scatter-Plots.ipynb

```diff
@@ -37,10 +37,8 @@
   },
```

Comment:
You would think so, but these scatter plots are not really that simple. <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=2>ReviewNB</a>_